Package:     fox1.6
Subject:     not compiling applications
Author:      Joachim Wiedorn <ad_debian at joonet.de>
Forwarded:   no
Last-Update: 2012-05-17

Inside the fox1.6 package is also the source code of some applications.
These programs only use the FOX library, but are not part of FOX libs.
Perhaps the could be moved to an extra Debian package - later.
---

diff -urN s08/configure.ac s09/configure.ac
--- s08/configure.ac	2012-05-17 16:06:58.099533663 +0200
+++ s09/configure.ac	2012-05-17 16:25:48.048819678 +0200
@@ -318,11 +318,5 @@
 doc/art/Makefile
 doc/screenshots/Makefile
 tests/Makefile
-adie/Makefile
-shutterbug/Makefile
-pathfinder/Makefile
-calculator/Makefile
-windows/Makefile
-fox.spec
 fox-config
 fox.pc)
diff -urN s08/Makefile.am s09/Makefile.am
--- s08/Makefile.am	2012-04-27 18:38:51.000000000 +0200
+++ s09/Makefile.am	2012-05-17 16:26:02.700827320 +0200
@@ -3,7 +3,7 @@
 # For non-gcc compilers, change the options to "no-dependencies"
 AUTOMAKE_OPTIONS = foreign dist-zip
 
-SUBDIRS    = utils include src chart doc tests shutterbug adie pathfinder calculator windows
+SUBDIRS    = utils include src chart doc tests
 EXTRA_DIST = ADDITIONS AUTHORS INSTALL LICENSE LICENSE_ADDENDUM TRACING README fox.lsm index.html aclocal.m4
 
 bin_SCRIPTS = fox-config
